src/doc/rustc/src/codegen-options/index.md

```diff
@@ -464,7 +464,15 @@ machine. Each target has a default base CPU.
 
 Individual targets will support different features; this flag lets you control
 enabling or disabling a feature. Each feature should be prefixed with a `+` to
-enable it or `-` to disable it. Separate multiple features with commas.
+enable it or `-` to disable it.
+
+Features from multiple `-C target-feature` options are combined. \
+Multiple features can be specified in a single option by separating them
+with commas - `-C target-feature=+x,-y`. \
+If some feature is specified more than once with both `+` and `-`,
+then values passed later override values passed earlier. \
+For example, `-C target-feature=+x,-y,+z -Ctarget-feature=-x,+y`
+is equivalent to `-C target-feature=-x,+y,+z`.
```

Comment:
Just out of curiosity, what's up with the `\` after periods? I see this is used elsewhere in this file, but have never encountered this Markdown syntax before.

Reply:
It's a line break.
I initially used trailing spaces for this, but it regularly confuses various people's editors, see https://github.com/rust-lang/rust/pull/71716#discussion_r418199810.

Reply:
Thanks, that makes sense! The choice of syntax for this is a bit confusing, as it has the exact opposite meaning in some programming languages like C...